src/lib/config-db.ts

```diff
@@ -96,9 +107,9 @@ export async function getConfig (logger: ComponentLogger): Promise<ConfigDb> {
     log('error loading config from db', err)
   }
 
```

Comment:
We probably just want an additional toggle that enables and disables recursive gateways.